### PR TITLE
Add CMD command to Kubefile

### PR DESCRIPTION
**Pull request by Stevent-fei**
### Describe what this PR does / why we need it
Temporary use of the CMD command.
ex:

Kubefile:
FROM docker.io/sealerio/kubernetes:v1.22.15
CMD kubectl get ns
CMD echo "AAAAAAA"
CMDS ["kubectl get node","kubectl get po -A"]

### CODE REVIEW

File: cmd/sealer/cmd/image/build.go [hunk 1]
```diff
@@ -74,6 +74,8 @@ func NewBuildCmd() *cobra.Command {
 		Args:    cobra.MaximumNArgs(1),
 		Example: exampleNewBuildCmd,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			logrus.Warn("CMD is about to be deprecated.")
```
Comment (kakaZhou719) [inline cmd/sealer/cmd/image/build.go:77]: add this warning message  at instruction process stage

File: build/kubefile/command/command.go [hunk 2]
```diff
@@ -22,6 +22,7 @@ const (
 	App    = "app"
 	Launch = "launch"
 	Cmds   = "cmds"
+	Cmd    = "cmd"
```
Comment (kakaZhou719) [inline build/kubefile/command/command.go:25]: we could add description to cmd

File: build/kubefile/parser/kubefile.go [hunk 3]
```diff
@@ -154,11 +162,22 @@ func (kp *KubefileParser) processOnCmd(result *KubefileResult, node *Node) error
...
 	case command.Cmds:
 		return kp.processCmds(node, result)
+	case command.Cmd:
+		return kp.processCmd(node, result)
 	default:
 		return fmt.Errorf("failed to recognize cmd: %s", cmd)
 	}
 }
 
+func (kp *KubefileParser) processCmd(node *Node, result *KubefileResult) error {
```
Comment (kakaZhou719) [inline build/kubefile/parser/kubefile.go:172]: here may could be simple, or could we just use processCmds?

File: build/kubefile/parser/kubefile.go [hunk 4]
```diff
@@ -133,6 +134,13 @@ func (kp *KubefileParser) generateResult(mainNode *Node) (*KubefileResult, error
 			if i != len(mainNode.Children)-1 {
 				return nil, errors.New("cmds should be the last instruction")
 			}
+
+		case command.Cmd:
+			cmdCnt++
+		}
+
+		if cmdCnt >= 1 && launchCnt == 1 {
```
Comment (kakaZhou719) [inline build/kubefile/parser/kubefile.go:142]: could we  do parameter judgment in  command.Cmd case

**Review by Starnop — COMMENTED** [review]
LGTM

**Review by Starnop — APPROVED** [review]

**Pull request merged**